public/xmlns/content.xsd

```diff
@@ -118,6 +118,29 @@
     </xs:simpleType>
 
     <!-- element nodes -->
+    <!-- Paragraph - Vertical stack of content with margin/padding on the top and bottom -->
+    <xs:complexType name="paragraph">
+        <xs:choice minOccurs="1" maxOccurs="unbounded">
+            <xs:group ref="content:elements" />
+        </xs:choice>
+    </xs:complexType>
+
+    <!-- Tabbed content -->
+    <xs:complexType name="tabs">
+        <xs:sequence>
+            <xs:element name="tab" minOccurs="1" maxOccurs="unbounded">
```

Comment:
yeah, we had talked about it, I'll go toss one on. We can always adjust it later :)